metadata/configuration/concepts/1215_hiv_testing_and_counseling_intake_form.csv

```diff
@@ -14,14 +14,23 @@ f2c680be-6efe-4bd1-97b8-48003c894d0d,,,"Operative Notes, Emergency","Notes opér
 b2e1ffa5-a6a8-4f3d-b797-25f11a66293b,,Yes full name;No full name;Do Not Know;Not Applicable,"HTC, Pregnancy Status","HTC, état de grossesse",Pregnancy Status,"HTC, état de grossesse",Misc,Coded,,
 a6df013d-8850-4b6f-b523-2d793aac0279,,,Sex Worker,Travailleur du sexe,Sex Worker,Travailleur du sexe,Misc,N/A,,
 762af91a-d2c1-4edc-9ec4-47a82c6f6485,,,People Who Inject Drugs,Pers. qui s'injectent des drogues,People Who Inject Drugs,Pers. qui s'injectent des drogues,Misc,N/A,,
-6d098664-6da4-4367-9037-9c8371c60f6e,,,MSM and Transgenders,HSH et transgenres,MSM and Transgenders,HSH et transgenres,Misc,N/A,,
+6d098664-6da4-4367-9037-9c8371c60f6e,,,MSM,HSH,MSM,HSH,Misc,N/A,,
 4982cf37-cb8a-4dea-bae8-002bf6162998,,,Blood or Organ Recipient,Recepteur de sang ou d'organe,Blood or Organ Recipient,Recepteur de sang ou d'organe,Misc,N/A,,
 3cbdb51b-86e9-4a44-802c-e99b033ad55d,,,Client of Sex Worker,Client de travailleuse du sexe,Client of Sex Worker,Client de travailleuse du sexe,Misc,N/A,,
 4b2379ba-a180-4f69-8802-493ed0177654,,,Migrant,Migrant,Migrant,Migrant,Misc,N/A,,
 e6f3cebd-aa29-4d88-8021-7949c55a65cd,,,Spouse/Partner of Migrant,Épouse / partenaire du migrant,Spouse/Partner of Migrant,Épouse / partenaire du migrant,Misc,N/A,,
-44861d4c-de67-4368-b5cf-279b754e5343,,Sex Worker;People Who Inject Drugs;MSM and Transgenders;Blood or Organ Recipient;Client of Sex Worker;Migrant;Spouse/Partner of Migrant;Others,"HTC, Risk Group","HTC, groupe de risque",Risk Group,"HTC, groupe de risque",Misc,Coded,,
+7b706d4b-b6c7-4383-ae60-de81151dfc2e,,,Transgenders,Transgenres,Transgenders,Transgenres,Misc,N/A,,
+44861d4c-de67-4368-b5cf-279b754e5343,,Sex Worker;People Who Inject Drugs;MSM;Transgenders;Blood or Organ Recipient;Client of Sex Worker;Migrant;Spouse/Partner of Migrant;Others,"HTC, Risk Group","HTC, groupe de risque",Risk Group,"HTC, groupe de risque",Misc,Coded,,
+be95b014-cf9e-45b4-a3f6-29d95d865f92,,,Clients of sex workers,Clients des travailleurs du sexe,Clients of sex workers,TransgClients des travailleurs du sexeenres,Misc,N/A,,
```

Comment:
@tresorluz there seem to be a copy/paste error on this line , please check the last part of this record (i.e. `TransgClients des travailleurs du sexeenres`)